modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -90,7 +133,7 @@ An additional role is required if the installation program is to create a vSpher
 |`Network.Assign`
 
 |Virtual Machine Folder
-|Always
+|Always.
```

Comment:
None of the other "Always" in the same category ("When required") have a period at the end.

Reply:
Thanks. Fixed.

Reply:
Thanks. Fixed.